### PR TITLE
[17.0][MIG] base_exception: Migration to 17.0

### PR DESCRIPTION
**Pull request by lef-adhoc**

### CODE REVIEW

**Review by matiasperalta1 — APPROVED** [review]

File: base_exception/wizard/base_exception_confirm_view.xml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+                <group>
+                    <field
+                        name="ignore"
+                        groups='base_exception.group_exception_rule_manager'
+                    />
+                </group>
+                <footer>
+                    <button
+                        name="action_confirm"
+                        string="_Close"
```
Comment (nguyenminhchien) [inline base_exception/wizard/base_exception_confirm_view.xml]: I think "_Close" is a typo.

File: base_exception/views/base_exception_view.xml [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+                                options="{'mode': 'python'}"
+                                placeholder="Enter Python code here. Help about Python expression is available in the help tab of this document."
+                            />
+                        </page>
+                        <page
+                            name="help"
+                            string="Help"
+                            invisible="exception_type != 'by_py_code'"
+                        >
+                            <group>
```
Comment (nguyenminhchien) [inline base_exception/views/base_exception_view.xml]: the view is broken, we should remove the `<group> `
image

**Review by nguyenminhchien — CHANGES_REQUESTED** [review]

File: base_exception/views/base_exception_view.xml (hunk 2 above)
Comment (lef-adhoc) [reply base_exception/views/base_exception_view.xml]: It's ready

File: base_exception/wizard/base_exception_confirm_view.xml (hunk 1 above)
Comment (lef-adhoc) [reply base_exception/wizard/base_exception_confirm_view.xml]: It's ready

**Review by nguyenminhchien — APPROVED** [review]

**Comment by john-herholz-dt** [comment]
@sebastienbeau , Can you merge this? I have several PRs depending on this. Thank you!

**Pull request merged**